prime-router/src/main/kotlin/Element.kt

```diff
@@ -792,86 +792,6 @@ data class Element(
         }
     }
 
-    /**
-     * The getDateTime function return the OffsetDatetime.  If it can't parse, it will throw either
-     * DateTimeParseException or DateTimeException.  Which allows the caller to catch the exception.
-     * @param [cleanedFormattedValue] datetime value to be parsed.
-     * @param [format] format to parse
-     * @return [OffsetDateTime] the best parsed datetime value
-     */
-    fun getDateTime(cleanedFormattedValue: String, format: String?): OffsetDateTime {
```

Comment:
Moving this method to a static scope allows it be more useful. The eventual home will be in the `DateUtils` that @mauricereeves PR is creating.